### PR TITLE
Add strawman spec

### PR DESCRIPTION
**Pull request by anssiko**
- Add strawman `index.bs` and generated `index.html`
- Rename explainer `README.md` -> `explainer.md`
- Add simple `README.md`

Fix https://www.w3.org/2020/04/16-webmachinelearning-minutes.html#a05

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Introduction {#intro}
+=====================
+
+For the introduction and use cases, please see the <a href="https://github.com/webmachinelearning/model-loader/blob/master/explainer.md">explainer.md</a>.
+
+API {#api}
+==========
+
+<pre class="idl">
+interface MLInference {
```
Comment (jbingham) [inline index.bs:22]: Should we rename this  interface to ModelLoader?

Comment (anssiko) [reply index.bs]: I do like that name better. PTAL https://github.com/webmachinelearning/model-loader/pull/2/commits/da354a67df3a78d41e609bb33a95e58cdaef3dd2

**Comment by anssiko** [comment]
@jbingham given you're now the editor of this spec, feel free to merge this PR when you feel good about it and think it has been adequately reviewed. The expectation is folks will start watching this repo and will drop review comments as needed. PR reviews are a collective responsibility of the whole CG, but you can @ selected folks who you'd like to review particular proposals based on their interest or expertise in the topic.

File: index.bs (hunk 1 above)
Comment (jbingham) [reply index.bs]: On further thought, maybe ModelLoader isn't explicit enough for a global name. MLModelLoader? Maybe the web will have AR/VR models.

Comment (huningxin) [reply index.bs]: Agree, WebXR may support their models. `MLModelLoader` is better.

Alternatively, we could group this interface under the Machine Learning (ML) entry point like `navigator.ml`? With that, the meaning of `ModelLoader` is explicit. Similar to [`navigator.xr`](https://www.w3.org/TR/webxr/#navigator-xr-attribute) or [`navigator.gpu`](https://gpuweb.github.io/gpuweb/#navigator-gpu).

The IDL sketch would be:
```webidl
partial interface Navigator {
  readonly attribute ML ml;
};

partial interface ML {
  ModelLoader createModelLoader();
};

interface ModelLoader {
  //TBD
};
```

The sample code of creating a `ModelLoader` object would change to:
```js
const modelLoader = navigator.ml.createModelLoader();
```

Any thoughts?

Comment (jbingham) [reply index.bs]: Great suggestion, Ningxin. We probably do want to follow this navigator.ml approach, so we can group any ML-related interfaces in a common namespace.

@anssiko if you agree, let's do it.

Comment (anssiko) [reply index.bs]: I updated the PR, PTAL https://github.com/webmachinelearning/model-loader/pull/2/commits/da57c8b0047d8b514d559e0851667bbcf2b96e7c

Here we only define the `ML` interface once and reference its definition from the WebNN spec, and then "augment" it with [partial interface](https://heycam.github.io/webidl/#dfn-partial-interface).

(We can cross-reference these two specs similarly for possible other reusable spec definitions going forward, [Bikeshed](https://tabatkins.github.io/bikeshed/) makes it quite easy.)

Comment (huningxin) [reply index.bs]: It looks good. Thanks @anssiko and @jbingham !

**Pull request merged**